### ISSUE TITLE
Training accuracy on Cifar10

### ISSUE THREAD

**Issue by zxl101** [BraveGump/CGDL-for-Open-Set-Recognition#8]
I've tried training on Cifar10 by only changing the training and validation dataset. The closed set accuracy I got is only at 0.788.  To get the result in Table 2 of the paper, do I need to change any hyperparameters? If possible, can you post the code for Table 2?

**Comment by guglielmocamporese** [comment]
I runned the code of this repo on other datasets and this what I obtained. And with this code I'm not able to reproduced the results of Table 2:

| Model | MNIST | SVHN | CIFAR10 | CIFAR+10 | CIFAR+50 | TinyImageNet |
| - | - | - | - | - | - | - |
| CGDL from the original paper [here](https://arxiv.org/pdf/2003.08823.pdf) | 0.994 | 0.935 | 0.903 | 0.959 | 0.950 | 0.762 | 
| CGDL from the released code | 0.989 | 0.845 | 0.671 | 0.724 | 0.713 | 0.651 |
